pkg/systemd/terminal.jsx

```diff
@@ -71,19 +79,28 @@ const _ = cockpit.gettext;
             if (this.state.channel)
                 terminal = (<Terminal ref="terminal"
                      channel={this.state.channel}
+                     theme={this.state.theme}
                      onTitleChanged={this.onTitleChanged} />);
             else
                 terminal = <span>Loading...</span>;
 
             return (
                 <div className="console-ct-container">
-                    <div className="panel-heading">
+                    <div className="panel-heading terminal-group">
                         <tt className="terminal-title">{this.state.title}</tt>
-                        <button ref="resetButton"
-                             className="btn btn-default pull-right"
-                             onClick={this.onResetClick}>{_("Reset")}</button>
+                        <div>
+                            <Select onChange={this.onThemeChanged}
+                                    id="theme-select"
+                                    initial={this.state.theme}>
+                                <SelectEntry data='dark'>{_("Dark")}</SelectEntry>
+                                <SelectEntry data='light'>{_("Light")}</SelectEntry>
+                            </Select>
+                            <button ref="resetButton"
+                                 className="btn btn-default"
+                                 onClick={this.onResetClick}>{_("Reset")}</button>
+                        </div>
                     </div>
-                    <div className="panel-body">
+                    <div className={["panel-body", this.state.theme].join(' ')}>
```

Comment:
Please leave an extra space after { and before }. Also, `"panel-body " + this.state.theme` is slightly simpler.